bin/gwd/gwd.ml

```diff
@@ -1230,12 +1230,16 @@ let make_conf from_addr request script_name env =
        {day = tm.Unix.tm_mday; month = succ tm.Unix.tm_mon;
         year = tm.Unix.tm_year + 1900; prec = Sure; delta = 0};
      today_wd = tm.Unix.tm_wday;
-     time = tm.Unix.tm_hour, tm.Unix.tm_min, tm.Unix.tm_sec; ctime = utm;
-     image_prefix =
+     time = tm.Unix.tm_hour, tm.Unix.tm_min, tm.Unix.tm_sec; ctime = utm
+    ; image_prefix =
```

Comment:
It is more of a stylistic question, but is it common practice in the code base to have semicolons at the beginning of the line in that case (record fields declaration) ? Because just by watching the previous lines it appears to not be the case in that particular segment.